principles/index.html

```diff
@@ -64,22 +55,58 @@ <h2>How This Document Fits In</h2>
+      <li>distress & intrusion</li> 
+      <li>relevance</li>
+      <li>reporting and context</li> 
+      <li>transparency</li>
+      <li>security</li>
+      <li>trust</li>
+      <li>explainability / comprehensibility</li> 
+      <li>competition / consolidation / centralization</li>
+      <li>inferences</li>
+      <li>identifiers</li>
+      <li>accountability</li>
+    </ul>
+
+    <p>Principles are organized in sections below regarding particular use cases or common concepts that apply across different use cases.</p>
+
     <section>
-      <h2>Consent Is Inappropriate</h2>
-      <p>
-        The Privacy Principles leave mention that people can consent to some kinds of data processing, but expresses
-        scepticism that consent is a valid mechanism to protect people's autonomy in a number of sitations.
-      </p>
-      <p>
-        Indeed, when the cross-context sharing of identifiers and of behavioural data is involved, the evidence
-        shows that no consent mechanism will support people in making autonomous choices.
-      </p>
-      <div class="issue">
-        <p>
-          Return to this with greater detail.
+      <h3>Measurement</h3>
+
+      <section>
+        <h4>Measurement should be private for safe, widespread usage, but always be under user control</h4>
+      </section>
+
+      <section>
+        <h4>Opting-out should not be visible</h4>
+        <p>Users may wish to opt-out of participation in measurement, but do so in such a way that is not visible to the sites they visit. Visible opt-out could lead to retaliation against, or coercion of, users who do not wish to participate in measurement.</p>
+      </section>
+
+      <section>
+        <h4>Measurement should not significantly enable cross-context recognition</h4>
+        
+        <p>Protections of differential privacy take the form of guarantees that aggregation or noise make participation in a particular measurement mostly indistinguishable, but also recognize that some information (as often quantified by parameters including epsilon) is released and could be combined with other known information to learn something with some (presumably very small) probability. "significantly" here is not yet detailed. The aggregated or noised measurement should not reasonably be usable to identify a particular user or to link an user's activity to another context.</p>
+
+        <p class="note">
+          Metrics to define significance are being evaluated by a separate task force.
         </p>
-      </div>
+
+        <p>Because measurement and attribution involve all kinds of viewing advertisements and a variety of other actions, in a wide range of different contexts, relying on understanding of, expectations about and consent over cross-context recognition as a result of ad measurement would be inappropriate.</p>
+      </section>
+
+      <section>
+        <h4>Measurement should not significantly enable inferences about individual people</h4>
```

Comment:
Is there a way to be more specific than "inferences"? It doesn't feel like a stretch to say all of these proposals allow some form of inference. If geo is a breakdown key, and I get a rough sense of conversion rate for that geo, then I can infer conversion rate for every individual in the geo

It feels like the intention behind most of these proposals is to prevent deterministic data collection on individuals and to prevent cross-site data collection

Reply:
It seems like a differential privacy protection, for example, would include limiting how much you can infer about whether I as an individual converted from measuring the conversion rate from my geographic region.

Deterministic data collection seems far too narrow a constraint: a site learning that I probably-but-not-definitely accessed a certain healthcare resource would be often unexpected and inappropriate, for example.

"significantly" is still open to be defined further -- like how high an increased probability or how individualized an inference might be.

Reply:
I think your language there may solve the problem. "Measurement should limit inferences about individual people" 

Your response also brings up another nuance though. You say "limiting how much you can infer about whether I as an individual converted from measuring the conversion rate from my geographic region" which speaks to making inference about past behavior. Is the restriction on inference scoped to past behavior?

Reply:
Hm if I make an inference about you _without even using your sensitive data_, is that a privacy violation? https://differentialprivacy.org/inference-is-not-a-privacy-violation/ argues that this is not a privacy violation and I agree. Additionally, it is an extremely difficult boundary to enforce (e.g. it prevents learning any sort of model of humanity or human nature - this is the "smoking causes cancer" example).

I think in part this section is getting at the "sensitive information disclosure" aspect of https://w3cping.github.io/privacy-threat-model/#hl-sensitive-information which is about the _explicit_ disclosure of the user's sensitive information.

Reply:
People may still have a privacy interest in what is learned about them even without use of their data, but I do think our focus here is on what can be inferred about an individual from their having participated in the measurement. That's the only kind of protection that differential privacy can afford, for example.

If that's unclear, I could add a sentence to the explanation here, or we could document elsewhere that the scope of the privacy principles about measurement is to participation in the measurement.

(I don't think that explicit disclosure is the only thing to protect against, and that's an old draft not currently under development.)

Reply:
I would appreciate a little bit more nuance added here, because as written it seems to directly forbid doing any kind of science on populations.